{"priority": "blocker", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "links": {"attachments": {"href": "data/repositories/osrf/srcsim/issues/173/attachments_page=1.json"}, "self": {"href": "data/repositories/osrf/srcsim/issues/173.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/srcsim/issues/173/watch"}, "comments": {"href": "data/repositories/osrf/srcsim/issues/173/comments_page=1.json"}, "html": {"href": "#!/osrf/srcsim/issues/173/walking-issues-with-update"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/srcsim/issues/173/vote"}}, "reporter": {"display_name": "Tyler Frasca", "uuid": "{ca580efe-cf6a-41bc-8ad2-c98164ba2429}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bca580efe-cf6a-41bc-8ad2-c98164ba2429%7D"}, "html": {"href": "https://bitbucket.org/%7Bca580efe-cf6a-41bc-8ad2-c98164ba2429%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/654ac1d15ac44e99fc460edd2880f234d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsTF-2.png"}}, "nickname": "tmfrasca5", "type": "user", "account_id": "557058:78b06f86-6933-4c74-bdde-fae32b4f1095"}, "title": "Walking issues with update", "component": {"name": "robot", "links": {"self": {"href": "data/repositories/osrf/srcsim/components/395698.json"}}}, "votes": 0, "watches": 2, "content": {"raw": "Within the past week, I updated the competition software on two machines and am having issues with walking. I am able to publish to the */ihmc_ros/valkyrie/control/footstep_list* topic (running 'rostopic echo' shows the data being published), however Valkyrie doesn't move.\r\n\r\nI have been testing the simulation using the provided *walk_test.py* script. I start the simulation with *roslaunch srcsim unique_task1.launch init:=true*. Then I run the *walk_test.py* script and usually Valkyrie will walk. However, if I try to run the script again in the same simulation instance, Valkyrie doesn't move. The msgs are being published to the topic, but are not executed. There are no errors published by the IHMC controller nor by ROS.\r\n\r\nWe have a team member who hasn't updated the competition software, and he is able to run the script many times in a single simulation. We are also able to successfully run our walking code.\r\n\r\nCurrently, we are unable to continue testing our walking code, because this is blocking us on two of our machines. Any information would be greatly appreciated!", "markup": "markdown", "html": "<p>Within the past week, I updated the competition software on two machines and am having issues with walking. I am able to publish to the <em>/ihmc_ros/valkyrie/control/footstep_list</em> topic (running 'rostopic echo' shows the data being published), however Valkyrie doesn't move.</p>\n<p>I have been testing the simulation using the provided <em>walk_test.py</em> script. I start the simulation with <em>roslaunch srcsim unique_task1.launch init:=true</em>. Then I run the <em>walk_test.py</em> script and usually Valkyrie will walk. However, if I try to run the script again in the same simulation instance, Valkyrie doesn't move. The msgs are being published to the topic, but are not executed. There are no errors published by the IHMC controller nor by ROS.</p>\n<p>We have a team member who hasn't updated the competition software, and he is able to run the script many times in a single simulation. We are also able to successfully run our walking code.</p>\n<p>Currently, we are unable to continue testing our walking code, because this is blocking us on two of our machines. Any information would be greatly appreciated!</p>", "type": "rendered"}, "assignee": null, "state": "resolved", "version": null, "edited_on": null, "created_on": "2017-05-19T00:29:36.156789+00:00", "milestone": null, "updated_on": "2017-05-26T01:00:45.181413+00:00", "type": "issue", "id": 173}